invenio_rdm_records/templates/records/record.html

```diff
@@ -0,0 +1,47 @@
+<!--
+    Copyright (C) 2019 CERN.
+    Copyright (C) 2019 Northwestern University, Galter Health Sciences Library & Learning Center.
+  
+    Invenio-RDM-Records is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+-->
```

Comment:
minor: this one is a Jinja2 template so we were good :D 

(change back if you want to send less bytes over the wire)